contracts/cw20-base/src/multitest/receiver_contract.rs

```diff
@@ -0,0 +1,139 @@
+use anyhow::{bail, Result as AnyResult};
+use cosmwasm_std::{
+    from_slice, Addr, DepsMut, Empty, Env, MessageInfo, Response, StdError, StdResult,
+};
+use cw_multi_test::{App, Contract, Executor};
+use sylvia::contract;
+use thiserror::Error;
+
+use super::receiver::{self, Receiver};
+
+pub struct ReceiverContract {}
```

Comment:
This should not be empty - all storage atoms should be part of this.

Reply:
for this contract there is no storage. I can rename it to ReceiverDummyContract maybe to show it is only for testing purposes